src/index.html

```diff
@@ -9,9 +9,169 @@
+          Surname:
+          <input
+            class="input"
+            type="text"
+            required
+            autocomplete="off"
+            id="Surname"
+          >
+        </label>
+        <label class="label" for="Name">
+          Name:
+          <input
+            class="input"
+            type="text"
+            required
+            autocomplete="off"
+            id="Name"
+          >
+        </label>
+        </label>
+        <label class="label" for="age">
+          How old are You?
+          <input
+            class="input"
+            type="number"
+            min="0"
+            max="100"
+            value="12"
+            id="age"
+          >
+        </label>
+        <label class="label" for="date">
+          Full date of birth:
+          <input
+            class="input"
+            type="date"
+            id="date"
+          >
+        </label>
+        <label class="label" for="term">
+          I accept the term of the agreement
+          <input
+            class="input radio"
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="label" for="email">
+          E-mail:
+          <input
+            class="input"
+            type="email"
+            placeholder="email@example.com"
+            id="email"
+          >
+        </label>
+        <label class="label" for="password">
+          Password:
+          <input
+            class="input"
+            type="password"
+            id="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label class="label" for="love-cats">
+          Do you love cats?
+          <input
+            class="input radio"
+            type="radio"
+            name="love-cats"
+            value="Yes"
+          >
+          Yes
+          <input
+            class="input radio"
+            type="radio"
+            name="love-cats"
+            value="No"
+          >
+          No
+        </label>
+        <label class="label" for="color">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </label>
+        <label class="label" for="time">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            id="time"
+          >
+        </label>
+        <label class="label" for="car-brands">
+          What are your favorite brand of cars?
+          <select
+            name="car-brands"
+            multiple
+            id="car-brands"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
not a must, but a rule-of-thumb is that values are usually lower-case to avoid issue with case-sensitivity when working with data on back-end.
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```